CodingGuidelines.md

```diff
@@ -0,0 +1,157 @@
+# Coding Guidelines
+## Project: innovativeproject-fantasy-battle-game
+Executives:
+* Kur, Wojciech
+* Korniewicz, Sebastian
+* Ormaniec, Wojciech
+* Seremak, Łukasz
+* Okrasa, Patryk
+
+## Intro
+All rules written **here**, have to be respected, to allow the review to be
+finished positively. In case of violation, of any rules written here, you can
+be requested to change the code adequately, and even **REVERTED**.
+
+## Include statements
+Every file should start with include section:
+
+```C#
+using UnityEngine;
+using ANY_OTHER_PACKAGE;
+
+// Down here your code
+```
+
+## Namespaces and file naming convention
+Every script file which have been created should be named, as _Namespace_
+it contains, and the namespace should be named with *PascalCase* rules.
+
+>Active file -> Map
+
+```C#
+using UnityEngine;
+
+namespace Map
+{
+  // Code which you want to put, goes inside
+}
+```
+
+## Whitespaces
+
+Every line should **never** end with whitespace. There are two main situations
+in which you can find commonly:
+
+```C#
+void someFunction(int x, int y)
+{
+  someFunction(
+    x,y
+    );
+
+    someFunction(x,y);
+}
+```
+
+in this case, you can notice TAB aligment on break, and space after `(` or `x,y`
+call. It is best to remove any other white spaces than `\n`.
+
+## Braces
+We shall use standard convention, which is both braces on new line. Which is:
+```C#
+void func()
+{
+	if(false)
+	{
+
+	}
+	else
+	{
+
+	}
+
+	class name
+	{
+
+	}
+}
+```
+
+with some exceptions. We will allow for one line definitions, `if` statements
+and initializers (props).
+
+```C#
+public void prop{get;set;}
+public int func()
+{
+	if(true){return 1;}
```

Comment:
whitespace between the condition and opened bracket?

Reply:
My bad